apps/callscreen/js/calls_handler.js

```diff
@@ -362,6 +362,18 @@ var CallsHandler = (function callsHandler() {
   }
   window.addEventListener('resize', updateAllPhoneNumberDisplays);
 
+  /**
+   * Return the number of calls currently present in one state or another.
+   * This includes all regular calls irrespective of their state plus a
+   * conference group call if one is present.
+   *
+   * @returns {Integer} The number of calls currently present.
+   */
+  function openLines() {
```

Comment:
Nit: call this ```numOpenLines()``` since one may mistake this for returning a ```bool``` without reading the comment here and function body.